.github/workflows/addDescriptiveLabels.js

```diff
@@ -0,0 +1,145 @@
+module.exports = async (github, context) => {
+  const issue = context.payload.issue;
+  const title = issue?.title?.toLowerCase();
+  if (!title) return;
+
+  const addLabels = async (labelsToAdd) => {
+    await github.rest.issues.addLabels({
+      issue_number: context.issue.number,
+      owner: context.repo.owner,
+      repo: context.repo.repo,
+      labels: labelsToAdd,
+    });
+  };
+
+  const labelsToAdd = [];
+  for (const component of components) {
+    if (title.includes(component.toLowerCase())) {
```

Comment:
Q: should we lowercase the title as well? 🤔 
e.g.: if I have an issue with title "ActivityIndicator is broken" it would not contains `activityindicator`, right?

Reply:
it's lowercased at L3 ;)

Reply:
Lorenzo is right, I just do it once when I initialize the title const